tests/sessions/claims/test_remove_claim.py

```diff
@@ -62,14 +71,16 @@ async def test_should_clear_previously_set_claim_using_handle(timestamp: int):
     s: SessionContainer = await create_new_session(dummy_req, "someId")
 
     payload = s.get_access_token_payload()
-    assert payload == {"st-true": {"v": True, "t": timestamp}}
+    assert payload["st-true"] == {"v": True, "t": timestamp}
 
     res = await remove_claim(s.get_handle(), TrueClaim)
     assert res is True
 
     session_info = await get_session_information(s.get_handle())
     assert session_info is not None
-    payload_after = session_info.access_token_payload
+    payload_after = (
+        session_info.access_token_payload
+    )  # FIXME: This doesn't have stuff other than userData?
```

Comment:
fixme

Reply:
It's actually about the `get_session_information` vs `session.access_token_payload` difference that we discussed. What did we decide to do about that inconsistency?

Reply:
Fixed. renamed to `custom_claims_in_access_token_payload`